lib/bpk-component-text-link/README.md

```diff
@@ -0,0 +1,51 @@
+# bpk-component-text-link
+
+> Backpack React Native text-link component.
+
+## Installation
+
+Check the main [Readme](https://github.com/skyscanner/backpack-react-native#usage) for a complete installation guide.
+
+## Usage
+
+```js
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { spacingBase } from 'bpk-tokens/tokens/base.react.native';
+import BpkTextLink from 'backpack-react-native/bpk-component-text-link';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: spacingBase,
+  }
+});
+
+export default () => (
+  <View style={styles.container}>
+    <BpkText>
+        Check out the website <BpkTextLink accessibilityLabel="link to website" onPress={() => {}}>here</BpkTextLink>
```

Comment:
My pet peeve is links that say `here`, links should be like road signs that tell you where they lead:

```suggestion
        Check out the <BpkTextLink accessibilityLabel="link to website" onPress={() => {}}>website</BpkTextLink>
```